### PR TITLE
Feat/offer section

### PR DESCRIPTION
**Pull request by wszczawinski**
- offer section update
- contact form styling update
- download files

### CODE REVIEW

File: src/template.html [hunk 1]
```diff
@@ -89,7 +92,9 @@
             </a>
             <div class="nav-list">
                 <a class="nav-list__item nav-about" href="#about-me">O mnie</a>
-                <a class="nav-list__item nav-teaching" href="#teaching">Nauczanie</a>
+                <!-- <a class="nav-list__item nav-teaching" href="#teaching">Nauczanie</a> -->
```
Comment (Kinga-Zawarczynska) [inline src/template.html:95]: commented code?

File: src/template.html [hunk 2]
```diff
@@ -190,52 +195,70 @@
...
+                    </div>
+
+                    <img src="images/cennik.png" alt="germani w liczbach" class="headers" />
 
-                   <img src="images/price.svg" alt="cena" class="price">
+                    <h3 class="offer-subtitle">Pliki do pobrania:</h3>
+
+                    <div class="download-buttons">
+                        <a class="btn" href="./Regulamin GermAni.pdf" target="_blank">Regulamin</a>
+                        <a class="btn" href="./Formularz zgłoszeniowy GermAni.docx" target="_blank">
```
Comment (Kinga-Zawarczynska) [inline src/template.html:253]: I think href cannot have blank spaces in the path

**Review by Kinga-Zawarczynska — CHANGES_REQUESTED** [review]

File: src/template.html (hunk 1 above)
Comment (wszczawinski) [reply src/template.html]: comments removed

File: src/template.html (hunk 2 above)
Comment (wszczawinski) [reply src/template.html:251]: works fine - no changes

**Pull request merged**